nodemapper/src/gui/Builder/components/Flow.tsx

```diff
@@ -115,10 +109,17 @@ export const wranglename = (name: string) => {
     .toLowerCase();
 };
 
-const ModuleNode = (props: NodeProps<ModuleData>) => {
+const ModuleNode = (props: NodeProps<NodeData>) => {
+  const nodeinfo = useAppSelector((state) => state.builder.nodeinfo);
+  let selected = false;
+  if (nodeinfo !== null && nodeinfo !== undefined && nodeinfo !== "") {
```

Comment:
Shouldn;t that be enough?
```suggestion
  if (nodeinfo) {
```